content/material_and_questions.md

```diff
@@ -44,4 +53,153 @@ In the below table, any minimum or maximum variation between material years are
+The entire question is part of the required question + answer.
+If a quizzer needs to provide the reference question, the provided reference question must: 
+- Be of the exact same type as the one on the card (CR, CVR, CRMA, CVRMA).  The quizmaster and/or answer judge must check to confirm this.
+- Have content AND wording is similar to the content and wording of the written question.
+    - For greater clarity, we have provided the following example: CVR According to Luke chapter 23: 37 King of what?  A. The Jews The quizzer can give the question “King of what?” or “King of whom?”, as Jews could be referring to either nation (what) or people (whom).  In this case the quizzer could also give “King of the what?”
+- Have the determining word or phrase from the quizmaster’s card. 
+    - In the 50/50 example, “What men?”, and “Lawless what?, the questions are not similar.
+The determination of a valid Reference question, and the valid sub-type only considers the content in the question. 
+
+##### Chapter-Verse
+Must have a single answer.
+The question must contain a word or phrase that exists more than once in the same chapter.
+Context is the single verse.
+The Question Type abbreviation is "CVR"
+
+##### Multiple-Answer, Chapter-Verse
+Must have more than one answer.
+The Question Type abbreviation is "MACVR"
+
+##### Chapter (only)
+The question must contain a word or phrase that exists more than once in the material.
+The Question Type abbreviation is "CR"
+
+##### Multiple-Answer, Chapter (only)
+Must have more than one answer.
+The Question Type abbreviation is "MACR"
+
+#### Quote
+Will be direct quotes from Scripture
+Must be quoted word-perfect.
+Must be strong enough to stand on its own without requiring additional verses to explain it.
+Must be spiritually significant.
+The entire reference is part of the required question + answer.
+Quizzers may go back and correct their answers if done in the 30-second time limit.  The quizzer will be required to make one full correct rotation. However, the quizzer can be called correct if they have made a mistake only in one area of the verse AND correct the mistake making it abundantly clear to the quiz master and answer judge that they know where their mistake was, and what he corrected words are.
+Context is defined as the verses or verses that are required.
+The quizzer may not continue any non-required verse or verses.
+
+##### Quote
+A single verse
+The Question Type abbreviation is "QT"
+
+##### Quote-these-Two-Verses
+Two consecutive verses
+The Question Type abbreviation is "Q2V"
+Should be verses that make more sense when quoted together.  Example: Gal 5:22-23 
+
+#### Finish
+Will be direct quotes from Scripture
+Must be quoted word-perfect
+Must be strong enough to stand on its own without requiring additional verses to explain it.
+Must be spiritually significant.
+The first five words must be unique to all Finish questions.
```

Comment:
Needs a rephrase to make clear that we're only comparing with other Finish questions of the same subtype, not "all Finish questions"

Reply:
Yes, thanks. 

Literally, the currently rulebook calls out FTV/FT/SIT. I THINK F2V/FTN are meant to be implied. 

I couldn't think of a great way to phrase this though. Because the "first five words must be unique to the question type" is applied differently:
* If F/Q list circulated, it applies to THAT universe of written Fs
* If no list circulated, it applies to the universe of "Valid Fs"

But since that seemed like a district-specific choice, I wasn't sure how to define it at the total rule book level.

Reply:
Useful for us to remember we can also solve some of these things with document architecture. For example, define a "Q/F" type, then "Q" as a sub-type thereof, then "Q2V" as a sub-type of the sub-type. And thus class theory lets the Q2V inherit everything defined in Q/F. In fact, a superclass of Q/F is "Question". :)

Reply:
Oh interesting. I like the idea of abstracting a Q/F type, since the first 4-5 bullets are repeated for each. 

Is there a way to make it clear which type levels are "types that are used for Type Distribution purposes" ??

Reply:
Sure. The type distribution is in a completely different section. And within that section, the types are targeted explicitly. So for example, you can define "Multiple Answer" as a distribution label and say it includes "MA" types explicitly, thus excluding MACR and MACVR. In other words, there's a type hierarchy in terms of concepts and rules, but the very bottom of that hierarchy are the actual explicit question types (EQTs). EQTs are what get listed on each row of the type distribution table. So for example: "Reference" questions are the combined EQTs of CR, CVR, MACR, MACVR. Stuff like that.

Reply:
I did run out of headers.... you can't have a 6 ###### header. So Q2V is NOT hierarchically below Q but instead at the same level. Would love ideas to allow me to place Q2V below Q hierarchically.

Reply:
Out of curiosity, what prevented a level 6 header? That should be completely OK. HTML has 6 levels. And with HTML5, you could have infinite, theoretically. Markdown should support it. That said, a subclass doesn't need to be a subheader. Could define MACR in parallel with MA and CR. Does that make sense? As in, define concepts together, then bottom-level types in parallel, but refer to superclasses from subclasses.

Reply:
![Screen Shot 2020-11-08 at 5 36 56 PM](https://user-images.githubusercontent.com/13351944/98486053-0a5e9780-21e9-11eb-9b61-d0decc7cc5a0.png)

Reply:
I don't think I follow your example. Don't I want MACR at a header below CR since I'm inheriting stuff from CR? Or have I accomplished that already since I explicitly said "Must meet all the requirements of a CR except the single answer requirement." ?

Reply:
No. In fact, you want MA, CR, and MACR to be in parallel header levels. But MACR inherits from MA and CR. There's document heirchy and inheritance, but as two separate things.